package.json

```diff
@@ -3,16 +3,30 @@
   "version": "2.9.1",
   "description": "Common utility functions to use with d3",
   "devDependencies": {
+    "d3": "^3.5.13",
     "ez-doc": "github:zambezi/ez-doc",
-    "grunt": "~0.4.1"
+    "jquery": "^2.2.1",
+    "karma": "^0.13.21",
+    "karma-chrome-launcher": "^0.1.7",
+    "karma-coverage": "^0.5.2",
+    "karma-mocha": "^0.1.3",
+    "karma-requirejs": "^0.2.5",
+    "karma-sinon-chai": "^0.1.6",
+    "mocha": "^2.4.5",
+    "requirejs": "^2.1.22",
+    "squirejs": "^0.2.1",
+    "underscore": "^1.8.3"
   },
-  "peerDependencies": {
+  "optionalDependencies": {
```

Comment:
It's not really optional though, is it? It should probably just be a dependency outright, or replaced altogether with XHR or something.